pyproject.toml

```diff
@@ -39,12 +39,14 @@ dependencies = [
     "atoti[aws,azure,gcp,jupyterlab,kafka,jdbc]==0.9.1",
     "pandas == 2.2.3",
     "jupyterlab <4.2",
-    "ruff >=0.6.9, <1"
+    "ruff >=0.6.9, <1",
+    "memory-profiler>=0.61.0",
+    "jupyter-resource-usage>=1.1.0",
```

Comment:
Added this when I was diagnosing Jupyter notebook memory usage, if this is out of scope though I can take it out. Otherwise, I think it's a nice add-on utility.

Reply:
Nice! Let's keep it.